test

```diff
@@ -1048,7 +1048,7 @@ test_rest_endpoints() {
     curl -s http://localhost:${port}/summary > output.json
 
     blockchain_length=$(cat output.json | jq -r .blockchainLength)
-    assert '100' $blockchain_length
+    assert '90' $blockchain_length
```

Comment:
Does this change make sense?

Reply:
Did this decrease by 10 because canonicity is factored into the "summary" somehow?

Reply:
This doesn't make sense... This value should be the best tip's `blockchain_length` which shouldn't have changed

Reply:
@Isaac-DeFrain So, perhaps we have a bug, and adding the new logs has helped discover it?

Reply:
Perhaps, something is definitely weird here. Adding more blocks at heights where we already have them shouldn't change the `blockchainLength` of the summary

Reply:
It's strange that we're seeing a change in the summary's `blockchainLength` but not a similar effect anywhere else.